experiments/AMIP/components/land/bucket_utils.jl

```diff
@@ -50,11 +50,11 @@ get_field(sim::BucketSimulation, ::Val{:area_fraction}) = sim.area_fraction
 get_field(sim::BucketSimulation, ::Val{:air_density}) = sim.integrator.p.bucket.ρ_sfc
 
 function update_field!(sim::BucketSimulation, ::Val{:turbulent_energy_flux}, field)
-    parent(sim.integrator.p.bucket.turbulent_energy_flux) .= parent(field)
+    parent(sim.integrator.p.bucket.turbulent_fluxes.shf) .= parent(field)
```

Comment:
@juliasloan25 @kmdeck Is it better if we pass latent and sensible heat fluxes separately to ClimaLand here?

Reply:
That would be better! Are we able to get them from Atmos?

Reply:
issue in Atmos: https://github.com/CliMA/ClimaAtmos.jl/issues/2630